websites/F/Flowlab/dist/presence.js

```diff
@@ -0,0 +1,55 @@
+const presence = new Presence({ clientId: "991160367629750372" }), flowlabTimestamp = Math.floor(Date.now() / 1000);
+presence.on("UpdateData", async () => {
+    const presenceData = {
+        largeImageKey: "icon",
+        startTimestamp: flowlabTimestamp,
+    };
+    switch (document.location.hostname) {
+        case "flowlab.io": {
+            switch (document.location.pathname) {
```

Comment:
You reference `document.location.pathname` many times, make this a variable and reference the variable instead. You should use:`{ pathname } = document.location` merged with the constant declaration for the PresenceData object

Reply:
Like this..? 
`const presenceData: PresenceData = {
		largeImageKey: "icon",
		startTimestamp: flowlabTimestamp,
      		{ pathname } = document.location
};`

Reply:
Nevermind, I got it.